src/main/java/com/khomishchak/cryptoportfolio/controllers/FeedbackController.java

```diff
@@ -0,0 +1,36 @@
+package com.khomishchak.cryptoportfolio.controllers;
+
+import com.khomishchak.cryptoportfolio.model.Feedback;
+import com.khomishchak.cryptoportfolio.model.requests.FeedbackRequest;
+import com.khomishchak.cryptoportfolio.services.FeedbackService;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/feedbacks")
+public class FeedbackController {
+
+    private final FeedbackService feedbackService;
+
+    public FeedbackController(FeedbackService feedbackService) {
+        this.feedbackService = feedbackService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Feedback> createFeedback(@RequestBody FeedbackRequest feedbackRequest) {
+        return new ResponseEntity<>(feedbackService.saveFeedback(feedbackRequest), HttpStatus.CREATED);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<Feedback>> getAllFeedback() {
+        return new ResponseEntity<>(feedbackService.getAllFeedbacks(), HttpStatus.OK);
```

Comment:
FYI: you can use `ResponseEntity.ok(feedbackService.getAllFeedbacks())`

Reply:
I have new ResponseEntity in all the controllers and wold like to be consistent about code style.